iaso/tests/api/test_storage.py

```diff
@@ -1011,7 +1048,7 @@ def test_get_logs_for_device_status_filter(self):
                 "created_at": 1580608922.0,
                 "storage_id": "EXISTING_STORAGE",
                 "storage_type": "NFC",
-                "status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
+                "storage_status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
```

Comment:
I believe that when status is "OK", the reason and comment should be null. Updated at should be set